src/bin/main.rs

```diff
@@ -1,8 +1,4 @@
-use actix_web::web::Data;
-use actix_web::{App, HttpServer};
-use anyhow::Context;
-use hilow::router::router;
-use hilow::state::State;
+use hilow::run::run;
```

Comment:
I think we could have a `server` module for this instead, `run` doesn't mean too much.
What do you think about defining a `server` module and have it export this function there?

So this would be:

```rust
use hilow::server::run
```